frontend/components/Dashboard/Post/Post.tsx

```diff
@@ -61,10 +61,11 @@ function buildPostOrderList(el: HTMLElement) {
   return postOrderList
 }
 
-function processSelectedTextArea(selection, parentElement: HTMLElement) {
+function processSelectedTextArea(selection: Selection, parentElement: HTMLElement) {
   const nodeList = buildPostOrderList(parentElement)
-  let startIdx = nodeList.indexOf(selection.baseNode)
-  let endIdx = nodeList.indexOf(selection.extentNode)
+
+  let startIdx = nodeList.indexOf(selection.anchorNode as Node)
```

Comment:
`baseNode` and `extentNode` were aliases for `anchorNode` and `focusNode`, which TypeScript recognizes as valid values. See [this StackOverflow](https://stackoverflow.com/a/33586253/2918915) for details